### DISCUSSION TITLE
Doubts about probe setting

### DISCUSSION THREAD

**Discussion by thecuress** [martin2250/OpenCNCPilot#193] [General]
hello..I'm using this type of probe (it's 12mm thick).....I use this for machining wood --- I have no problem probing...but if I then apply the heightmap the cutter doesn't touch the surface when cutting....thank you very much for your reply

![616r2xk+U+L _AC_SX679_](https://github.com/user-attachments/assets/c181eb8c-512f-4589-a3be-b9e5e780ae55)

**Comment by deHarro** [comment]
Hi!
Can you please elaborate shortly on how you use this probe for generating a hight map?
Thanks!
Harry

**Reply by deHarro** [reply]
Perhaps you only have to initially (before starting the height map process) set the origin to Z=zero with the probe placed there (at the origin).

As so:

G38.2Z-10F20
G92Z0

or 

G38.2Z-10F50
G92Z0
$J=G91F100Z1.0

This is the code I personally use at my router.

The last command $J=... is a jog command and retracts the bit 1 mm above the probe to be freely moved in X/Y direction, if that is wanted.
The jog commands ignore the alarm state which is issued by the probing act.

**Comment by thecuress** [comment]
Excellent.... works perfectly thank you very much !!!
